### PR TITLE
Create Tests for Url Repository File Impl in test/storage package

### PR DESCRIPTION
**Pull request by Pinkavsky**
Make tests in UrlRepositoryFileImplTest:
- shouldCreateAlias
- serializesOneUrl
- shouldDeleteUrl
- shouldGetAllAliasesForUser
and other.

### CODE REVIEW

**Review by Narberial — APPROVED** [review]

**Review by IDavydenko21 — APPROVED** [review]

File: src/test/java/edu/kpi/testcourse/storage/UrlRepositoryFileImplTest.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    ).contains(alias, email, destinationUrl);
+  }
+
+  //Должен создаться Alias.
+  @Test
+  void shouldCreateAlias() {
+    UrlAlias url = new UrlAlias("http://r.org/short", "http://g.org/long", "user1@example.org");
+    urlRepository.createUrlAlias(url);
+    assertThat(urlRepository.findUrlAlias("http://r.org/short")).isEqualTo(url);
+  }
```
Comment (Pinkavsky) [inline src/test/java/edu/kpi/testcourse/storage/UrlRepositoryFileImplTest.java:65]: Should create alias

File: src/test/java/edu/kpi/testcourse/storage/UrlRepositoryFileImplTest.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  @Test
+  void shouldDeleteUrl() {
+    UrlAlias url = new UrlAlias("http://r.org/short", "http://g.org/long", "user1@example.org");
+    urlRepository.createUrlAlias(url);
+    urlRepository.deleteUrlAlias("user1@example.org", "http://r.org/short");
+    assertThat(urlRepository.findUrlAlias("http://r.org/short")).isNull();
+    assertThat(urlRepository.getAllAliasesForUser("user1@example.org"))
+      .asList()
+      .doesNotContain(url);
+  }
```
Comment (IDavydenko21) [inline src/test/java/edu/kpi/testcourse/storage/UrlRepositoryFileImplTest.java:77]: Should delete url

File: src/test/java/edu/kpi/testcourse/storage/UrlRepositoryFileImplTest.java [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    String alias = "http://r.org/short";
+    String email = "user1@example.org";
+    String destinationUrl = "http://g.org/long";
+    UrlAlias url = new UrlAlias(alias, destinationUrl, email);
+    urlRepository.createUrlAlias(url);
+    Assertions.assertThat(
+      Files.readString(appConfig.storageRoot()
+        .resolve("url-repository.json"), StandardCharsets.UTF_8)
+    ).contains(alias, email, destinationUrl);
+  }
```
Comment (Pinkavsky) [inline src/test/java/edu/kpi/testcourse/storage/UrlRepositoryFileImplTest.java:57]: serializes one url.

File: src/test/java/edu/kpi/testcourse/storage/UrlRepositoryFileImplTest.java [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    UrlAlias URL3 = new UrlAlias("http://r.org/short3", "http://g.org/long3", "user1@example.org");
+    UrlAlias URL4 = new UrlAlias("http://r.org/short4", "http://g.org/long4", "gagauser1@example.org");
+    urlRepository.createUrlAlias(URL1);
+    urlRepository.createUrlAlias(URL2);
+    urlRepository.createUrlAlias(URL3);
+    urlRepository.createUrlAlias(URL4);
+    List<UrlAlias> urls = urlRepository.getAllAliasesForUser("user1@example.org");
+    assertThat(urls.size()).isEqualTo(3);
+    assertThat(urls).asList().contains(URL1, URL2, URL3);
+  }
```
Comment (Narberial) [inline src/test/java/edu/kpi/testcourse/storage/UrlRepositoryFileImplTest.java:93]: Should get all aliases for user

**Pull request merged**